src/redux/Books/books.js

```diff
@@ -0,0 +1,20 @@
+import { createReducer, createAction } from '@reduxjs/toolkit';
+
+export const ADD_BOOK = createAction('books/add');
+export const REMOVE_BOOK = createAction('book/remove');
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure that the actions follow the duck pattern.

- As an example the `ADD_BOOK` action could look more like the following:
```javascript
ADD_BOOK = 'bookstore/books/ADD_BOOK'
```

Reply:
@ShadyShawkat i am not clear with this please do explain further

Reply:
Hey @Dmambo,

The [duck pattern](https://github.com/erikras/ducks-modular-redux) should look like the following when typing the action types:

```javascript
const ACTION = '<npm-module-or-app/reducer/ACTION_TYPE>'
```

The requested change here is in regard to the string used for every action type